assets/css/main.css

```diff
@@ -14,6 +15,7 @@
   --bg-light-color-3: #bfccf8;
   --bg-light-color-4: #7089db;
   --button-red: #c32222;
+  --dark-blue: #001a72;
```

Comment:
We already have variable for that color, it's `--font-dark-color`

Reply:
Changed that